sdk/eventgrid/azure-messaging-eventgrid-systemevents/pom.xml

```diff
@@ -14,11 +14,11 @@ Code generated by Microsoft (R) TypeSpec Code Generator.
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventgrid-systemevents</artifactId>
-  <version>1.0.0-beta.3</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid-systemevents;current} -->
+  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid-systemevents;current} -->
   <packaging>jar</packaging>
 
-  <name>Microsoft Azure SDK for System Events</name>
-  <description>This package contains Microsoft Azure System Events client library.</description>
+  <name>Microsoft Azure SDK for Event Grid System Events</name>
+  <description>This package contains strongly-typed model classes for Azure Event Grid System Events.</description>
```

Comment:
I see this package has a dependency on `azure-core-http-netty`. This is a models-only library. So, we should not have this dependency.

Reply:
Also, we have azure-xml dependency. Is that required?